src/item/VanillaCooldownTags.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+/**
+ * Tags used by items to determine their cooldown group.
+ * @see Item::getCooldownTag()
+ */
+final class VanillaCooldownTags{
+
+	private function __construct(){
+		//NOOP
+	}
+
+	public const CHORUS_FRUIT = "chorus_fruit";
+	public const ENDER_PEARL = "ender_pearl";
+	public const SHIELD = "shield";
+	public const GOAT_HORN = "goat_horn";
+}
```

Comment:
It should be documented that:
> These tag values are not related to Minecraft internal IDs.
They only share a visual similarity because these are the most obvious values to use.
Any arbitrary string can be used.

Otherwise people might get confused and want to link them with the network IDs, and we really don't want that (no protocol stuff in the API).

(It's also possible we could make an interface and require people to use unbacked enums, but strings are easier).